supervisor/shared/safe_mode.c

```diff
@@ -28,7 +28,9 @@
 
 #include "mphalport.h"
 
+#if defined(MICROPY_HW_LED_STATUS)
 #include "shared-bindings/digitalio/DigitalInOut.h"
+#endif
```

Comment:
This `#if` is breaking the esp32-s2 builds, I guess because `MICROPY_HW_LED_STATUS` didn't get defined for those builds.